CHANGELOG.md

```diff
@@ -52,7 +52,6 @@ All notable, unreleased changes to this project will be documented in this file.
 - Add `withChoices` flag for Attribute type - #7733 by @dexon44
 `CheckoutAddPromoCode`, `CheckoutPaymentCreate` will raise a ValidationError when product in the checkout is
 unavailable - #8978 by @IKarbowiak
-- Fix crash when Avalara plugin was used together with Webhooks plugin for shipping methods - #9121 by @rafalp
```

Comment:
🚔

Reply:
This was a bad merge, it's already in line 58